### PR TITLE
[nodes] new SfMTriangulation node

### PR DESCRIPTION
**Pull request by mh0g**
## Description

Adds a main that is essentially the triangulation step from the sequential SfM.
AliceVision PR: https://github.com/alicevision/AliceVision/pull/1314

## Features list
This allows to run the triangulation without going into the sfm step.

### CODE REVIEW

**Comment by mugulmd** [comment]
When testing this node it kept failing with the following error: 
`ERROR: unrecognised option '--maxMemory=9223372036854771712'`

**Comment by mugulmd** [comment]
Tested on 3 datasets by using the output of the `StructureFromMotion` node as input of the `Triangulate` node. 
Results: 
- the generated landmarks are similar to the ones generated by `StructureFromMotion`, with a few differences (which is expected since there is no bundle adjustment)
- when increasing the `minNumberOfObservationsForTriangulation` the number of landmarks decreases (which is expected).

I also ran the following sanity check: when using an `SfMData` without poses as input, no landmark is generated.

**Comment by mh0g** [comment]
> Tested on 3 datasets by using the output of the `StructureFromMotion` node as input of the `Triangulate` node. Results:
> 
>     * the generated landmarks are similar to the ones generated by `StructureFromMotion`, with a few differences (which is expected since there is no bundle adjustment)
> 
>     * when increasing the `minNumberOfObservationsForTriangulation` the number of landmarks decreases (which is expected).
> 
> 
> I also ran the following sanity check: when using an `SfMData` without poses as input, no landmark is generated.

Tested on windows, seems to work as intended.
Tested on a benchmark dataset with ground truth poses and features/matches from meshroom. Small differences vs running with poses from SfM (as expected). Can also confirm that the landmarks are recomputed when using the ouptut of the Sfm node.

File: meshroom/nodes/aliceVision/Triangulate.py [hunk 1]
```diff
@@ -0,0 +1,164 @@
+__version__ = "1.0"
+
+from meshroom.core import desc
+
+
+class Triangulate(desc.AVCommandLineNode):
```
Comment (fabiencastan) [inline meshroom/nodes/aliceVision/Triangulate.py:6]: Triangulation

**Comment by fabiencastan** [comment]
It would be good to also add a node for https://github.com/alicevision/AliceVision/blob/develop/src/software/pipeline/main_computeStructureFromKnownPoses.cpp
Or fuse them in a single node.

**Comment by fabiencastan** [comment]
After checking, no need to integrate computeStructureFromKnownPoses, already possible with FeatureMatching+Triangulation.

**Review by fabiencastan — APPROVED** [review]

**Review by fabiencastan — APPROVED** [review]

**Pull request merged**